src/pages/blog/article_preview.jsx

```
@@ -0,0 +1,121 @@
+        <div className="text-2xl font-thin text-neutral-800 dark:text-neutral-100">
+          {formData.category}
+        </div>
+        <img
+          src={formData.image}
+          alt="Article Image"
+          className="w-full rounded-lg mt-2 max-h-[500px] object-cover"
+        />
+
+        <div className="mt-4 text-base text-neutral-600 dark:text-neutral-300">
+        <div
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)